Lib/os.py

```diff
@@ -427,6 +427,13 @@ def _walk(top, topdown, onerror, followlinks):
 __all__.append("walk")
 
 if {open, stat} <= supports_dir_fd and {scandir, stat} <= supports_fd:
+    from enum import Enum, _simple_enum
+
+    @_simple_enum(Enum)
+    class _WalkAction:
+        YIELD = 1
+        CLOSE = 2
+        WALK = 3
```

Comment:
This might lead to performance regressions, since `enum` is known to have slow import times, and `os` is quite a low-level module that's imported by a lot of other modules.

In this case, it looks like this probably doesn't _need_ to be an enum. You could maybe achieve the same thing just like this:

```suggestion
    class _WalkAction:
        YIELD = object()
        CLOSE = object()
        WALK = object()
```

Reply:
Yeah, good call - thanks!